op-node/rollup/sync/config.go

```diff
@@ -63,7 +63,9 @@ func (m *Mode) Clone() any {
 
 type Config struct {
 	// SyncMode is defined above.
-	SyncMode Mode `json:"syncmode"`
+	SyncMode        Mode `json:"syncmode"`
+	SyncModeReqResp bool `json:"syncmode_reqresp"`
```

Comment:
Why don't we just use the shorter version
```suggestion
	ReqResp bool `json:"reqresp"`
```
? It's a field of the `sync.Config` so from context it's clear what it's about.

`SyncModeReqResp` seems overly verbose to me--and in a way even misleading, because `SyncMode` is more about inital sync while req-resp is about unsafe gap sync.

Reply:
`SyncMode` also governs if we will be using the payloads queue (i.e. the internal cache) - there is some difference there between ELSync and CLSync.

I am opting to not just use ReqResp here, because it is really an overloaded symbol.

Ideally both of these should be removed in a future milestone, where we consolidate CLSync and ELSync modes.

I'd rather we keep it as is, as now it is very clear it refer to `SyncMode` in some sense.